src/components/card/card.tsx

```diff
@@ -161,7 +164,12 @@ export type OuiCardProps = Omit<CommonProps, 'aria-label'> &
      * **DEPRECATED: Use `betaBadgeProps.title` instead.**
```

Comment:
do you want to update this comment, too?

```suggestion
     * **DEPRECATED: Use `experimentalBadgeProps.title` instead.**
```